### PR TITLE
Swift Style selectors + Option to change Font size

### PR DESCRIPTION
**Pull request by kshdeo**

### CODE REVIEW

**Comment by cezarywojcik** [comment]
Thanks for the format update!
Per the font size, however, I do want to keep parity between the ObjC and Swift versions here. In ObjC, the font size is a `#define` constant. In addition, the font size can change at any time, so there needs to be an assurance that reapplying the font size is handled properly.

Please remove the font size change for now, and I'll merge this in :)

**Comment by kshdeo** [comment]
Ah, I agree for maintaining parity with Objective-C. Will probably handle that in a later pull request. For now, reverting the changes to font size

File: CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift [hunk 1]
```diff
@@ -460,13 +460,12 @@ public class CWStatusBarNotification : NSObject {
...
         NSNotificationCenter.defaultCenter().addObserver(self,
-            selector: "updateStatusBarFrame",
+            selector:#selector(CWStatusBarNotification.updateStatusBarFrame),
             name: UIApplicationDidChangeStatusBarFrameNotification,
             object: nil)
         
         // checking for status bar change
-        NSNotificationCenter.defaultCenter().addObserver(self,
-            selector: "updateStatusBarFrame",
+        NSNotificationCenter.defaultCenter().addObserver(self, selector:#selector(CWStatusBarNotification.updateStatusBarFrame),
```
Comment (cezarywojcik) [inline CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift]: Please break up this line like the other notification center bits of code above. Also, please add a space after `selector:`.

File: CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift [hunk 2]
```diff
@@ -385,13 +385,13 @@ public class CWStatusBarNotification : NSObject {
             
             // checking for screen orientation change
             NSNotificationCenter.defaultCenter().addObserver(self,
-                selector: "updateStatusBarFrame",
+                selector:#selector(CWStatusBarNotification.updateStatusBarFrame),
```
Comment (cezarywojcik) [inline CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift]: Please add a space after `selector:`.

File: CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift [hunk 3]
```diff
@@ -385,13 +385,13 @@ public class CWStatusBarNotification : NSObject {
...
                 name: UIApplicationDidChangeStatusBarFrameNotification,
                 object: nil)
             
             // checking for status bar change
             NSNotificationCenter.defaultCenter().addObserver(self,
-                selector: "updateStatusBarFrame",
+                selector:#selector(CWStatusBarNotification.updateStatusBarFrame),
```
Comment (cezarywojcik) [inline CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift]: Please add a space after `selector:`.

File: CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift [hunk 4]
```diff
@@ -460,13 +460,12 @@ public class CWStatusBarNotification : NSObject {
         
         // checking for screen orientation change
         NSNotificationCenter.defaultCenter().addObserver(self,
-            selector: "updateStatusBarFrame",
+            selector:#selector(CWStatusBarNotification.updateStatusBarFrame),
```
Comment (cezarywojcik) [inline CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift]: Please add a space after `selector:`.

**Comment by cezarywojcik** [comment]
Added some comments; will merge this in once those are addressed. Thanks!

**Comment by kshdeo** [comment]
Done

**Comment by cezarywojcik** [comment]
Thanks!

**Pull request merged**